tests/dialects/test_riscv.py

```diff
@@ -89,6 +90,42 @@ def test_comment_op():
     assert code == "    # my comment\n"
 
 
+def test_label_op_without_comment():
+    label_str = "mylabel"
+    label_op = riscv.LabelOp(label_str)
+
+    assert label_op.label.data == label_str
+
+    code = riscv_code(ModuleOp([label_op]))
+    assert code == f"{label_str}:\n"
+
+
+def test_label_op_with_comment():
+    label_str = "mylabel"
+    label_op = riscv.LabelOp(f"{label_str}", comment="my label")
+
+    assert label_op.label.data == label_str
+
+    code = riscv_code(ModuleOp([label_op]))
+    assert code == f"{label_str}:                                         # my label\n"
+
+
+def test_label_op_with_region():
+    @Builder.implicit_region
+    def label_region():
+        a1_reg = TestSSAValue(riscv.RegisterType(riscv.Registers.A1))
+        a2_reg = TestSSAValue(riscv.RegisterType(riscv.Registers.A2))
+        riscv.AddOp(a1_reg, a2_reg, rd=riscv.Registers.A0)
+
+    label_str = "mylabel"
+    label_op = riscv.LabelOp(f"{label_str}", region=label_region)
+
+    assert label_op.label.data == label_str
+
+    code = riscv_code(ModuleOp([label_op]))
+    assert code == f"{label_str}:\n    add a0, a1, a2\n"
+
+
```

Comment:
There are still changes from the label op in this PR, or am I mistaken?

Reply:
yeah, I'd suggest rebasing on the `main` branch since that PR is now merged.